src/Tests/Singularity.Test/Registrations/ContainerBuilderTests.cs

```diff
@@ -171,6 +171,24 @@ public void GetDependencies_MultiDecoratorRegistration_Enumerate()
             Assert.Equal(new[] { typeof(PluginLogger1), typeof(PluginLogger2), typeof(PluginLogger3) }, decorators.Select(x => x.Type));
         }
 
+        [Fact]
+        public void GetDependencies_RegisterModule()
+        {
+            //Arrange
```

Comment:
For consistently reasons please name this `ARRANGE`

Reply:
I apologize for this. I try to be as consistent as possible when submitting code to someone's body of work. I must have missed this, or didn't think much of it at the time. At any rate, I have submitted changes to address this.

Reply:
No problem, your code is pretty clean and its easy to miss tiny things.